### ISSUE TITLE
Add ability for users to download a snapshot of the current slide

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2690]
Originally reported on Google Code with ID 1942

```
Suggested in this thread

  https://groups.google.com/d/msg/bigbluebutton-dev/ldWgkOkY3G4/eh7VjgCs9C4J

This is also related with giving the users the ability to download the presentation
(with annotations), see

   https://code.google.com/p/bigbluebutton/issues/detail?id=285

```

Reported by `ffdixon` on 2015-05-12 15:01:36

**Comment by GhaziTriki** [comment]
The original thread is:

> By default, BigBlueButton does not give the viewers any link to download the presentation.
> 
> Possible scenarios:
> 
> 1. Provide the download link automatically when the presenter has uploaded the slides
> 
> 2. Provide the download link only if the presenter has granted permissions for download. There could be a check box during the upload process.
> 
> 3. Provide the user a list of download links after they logout (this keeps the presenter in control of the current slides during the presentation).

All features are implemented to 2.1 except point 3.

**Comment by antobinary** [comment]
Exists on BBB 2.5+
![Screenshot from 2023-02-14 15-34-31](https://user-images.githubusercontent.com/6312397/218856198-85101bd1-c647-4c29-9fbd-cf44d7fd3f69.png)
